ACadSharp/IO/Templates/CadDimensionStyleTemplate.cs

```diff
@@ -32,17 +32,17 @@ public override void Build(CadDocumentBuilder builder)
 				this.CadObject.Style = style;
 			}
 
-			if (builder.TryGetCadObject(this.DIMLDRBLK, out Block leaderArrow))
+			if (builder.TryGetCadObject(this.DIMLDRBLK, out BlockRecord leaderArrow))
```

Comment:
For this kind of operations I've implemented a method to get a table entry and simplify the logging in case is not found, check `CadTemplate.getTableReference` you can use it to retrieve the `BlockRecord` in case of a not found it will notify to the user.